packages/internal-helpers/src/path.ts

```diff
@@ -104,3 +104,9 @@ export function removeBase(path: string, base: string) {
 	}
 	return path;
 }
+
+const withFileExt = /\/[^/]+\.\w+$/;
+
+export function hasFileExtension(path: string) {
+	return withFileExt.test(path);
```

Comment:
```suggestion
const WITH_FILE_EXT = /\/[^/]+\.\w+$/;

export function hasFileExtension(path: string) {
	return WITH_FILE_EXT.test(path);
```

Reply:
We already have the `fileExtension`, so can't we use that? The result would be an empty string 🤔

Reply:
Could do, but `fileExtension` will need a rewrite to use regex and not split though, because it isn't robust in cases where there's no extension. A dot anywhere in the path will give a false positive.

Reply:
I don't think this would be the right approach. It's running on every request, so I don't think it makes sense to use a function that does more than is needed.